### PR TITLE
query: Always return resp.json() as long as there's no exception

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by JOJ0** [comment]
thanks, again: very happy you found this bug! :-)

File: synadm/api.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ def query(self, method, urlpart, params=None, data=None):
                 params=params, json=data
             )
             resp.raise_for_status()
-            if resp.ok:
-                return resp.json()
-            self.log.warning("No valid response from Synapse")
+            if not resp.ok:
```
Comment (rht) [inline synadm/api.py:42]: @JOJ0 actually the `resp.raise_for_status()` above raises an exception even before `resp.json()` got returned. Should we remove `resp.raise_for_status()`?

Comment (rht) [reply synadm/api.py:42]: #20

Comment (JOJ0) [reply synadm/api.py:42]: Ok I understand but didnt you test this change before submitting the PR??

Comment (rht) [reply synadm/api.py:42]: Sorry, I did test a version of this change, but when I recreated the change in this PR (#19), I somehow missed that there is the `resp.raise_for_status()` line. Let's say it was a lucky accident that I was able to see the failed response json at all, that I somehow put `resp.json()` before `resp.raise_for_status()`.

**Pull request merged**